src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAdditionalInfo.tsx

```diff
@@ -42,7 +42,8 @@ export class ArtworkDetailsAdditionalInfo extends React.Component<
       {
         title: "Condition",
         value:
-          isBiddable && sd.ENABLE_REQUEST_CONDITION_REPORT ? (
+          canRequestLotConditionsReport &&
+          sd.ENABLE_REQUEST_CONDITION_REPORT ? (
```

Comment:
note that we have an [isomorphic `getENV` util](https://github.com/artsy/reaction/blob/master/src/Utils/getENV.tsx) that we can use to check ENV vars safely on the client and server.